analyzer/consensus/genesis.go

```diff
@@ -296,7 +295,7 @@ func (mg *GenesisProcessor) addGovernanceBackendMigrations(batch *storage.QueryB
 				proposal.State.String(),
 				proposal.Deposit.ToBigInt(),
 				proposal.Content.ChangeParameters.Module,
-				hex.EncodeToString(proposal.Content.ChangeParameters.Changes),
+				proposal.Content.ChangeParameters.Changes,
```

Comment:
This was actually broken since https://github.com/oasisprotocol/nexus/pull/630 

The old version had to use `hex.EncodeToString` with `decode('%s', 'hex')` since it was working with strings. https://github.com/oasisprotocol/nexus/pull/630/files#diff-dad00025ae7b123664532df3771612a4339161128cb7de77fcc18af2bfa122faL513-L521

Probably none of the deployments had a reindex done which since then, so it is probably ok on all live deployments.